### DISCUSSION TITLE
Filter by subset

### DISCUSSION THREAD

**Discussion by shabeerak** [milvus-io/milvus#34822] [Q&A and General discussion]
Hi,
I see Milvus supports filtering array filtering with contains all. 
`filter="ARRAY_CONTAINS_ALL(color_coord, [7, 8])"`

Is there a way to filter if **color_coord** is a subset of an input array filter. 
Say, all elements of **color_coord** is a member of the input array filter?

thanks!

**Comment by yhmo** [comment]
No such operator.

**Comment by xiaofan-luan** [comment]
can't you do ARRAY_CONTAINS_ALL([7, 8], color_coord)?

**Reply by yhmo** [reply]
Will get an error: "cannot parse expression: ARRAY_CONTAINS_ALL([7], arr), error: contains_all operation are only supported on json or array fields now, got: [7]: invalid parameter"

**Comment by xiaocai2333** [comment]
Currently, filtering with a field as a subset is not supported, an issue has been logged: #35134 .
